src/Compilers/Core/Portable/Syntax/SyntaxListBuilderExtensions.cs

```diff
@@ -42,7 +44,7 @@ public static SyntaxList<TNode> ToList<TNode>(this SyntaxListBuilder builder)
                 return default(SeparatedSyntaxList<TNode>);
             }
 
-            return new SeparatedSyntaxList<TNode>(new SyntaxNodeOrTokenList(builder.ToListNode().CreateRed(), 0));
+            return new SeparatedSyntaxList<TNode>(new SyntaxNodeOrTokenList(builder.ToListNode()!.CreateRed(), 0));
```

Comment:
💡 Consider refactoring this method:

```
var listNode = builder.ToListNode();
if (listNode is null)
{
  return null;
}

return new SeparatedSyntaxList<TNode>(new SyntaxNodeOrTokenList(listNode.CreateRed(), 0));
```

Applies to methods above too.

Reply:
That would change the return type of the method to be a nullable struct. That's not a trivial change here. 

---
In reply to: [362969063](https://github.com/dotnet/roslyn/pull/40729#discussion_r362969063) [](ancestors = 362969063)

Reply:
Did u mean `return default`?

---
In reply to: [362974659](https://github.com/dotnet/roslyn/pull/40729#discussion_r362974659) [](ancestors = 362974659,362969063)

Reply:
Yes I meant `return default`